source/wp-content/themes/wporg-showcase-2022/templates/search.html

```diff
@@ -1,37 +1,37 @@
 <!-- wp:template-part {"slug":"header" } /-->
 
 <!-- wp:group {"tagName":"main","style":{"spacing":{"blockGap":"0px"}},"className":"entry-content","layout":{"type":"constrained"}} -->
-<main class="wp-block-group entry-content">
-	<!-- wp:query-title {"type":"search"} /-->
-	<!-- wp:pattern {"slug":"wporg-showcase-2022/secondary-search-bar"} /-->
+<main class="wp-block-group entry-content"><!-- wp:group {"align":"wide","layout":{"type":"default"}} -->
+	<div class="wp-block-group alignwide">
+		<!-- wp:pattern {"slug":"wporg-showcase-2022/results-bar"} /-->
```

Comment:
This is the only change that really matters.

The other layout change help the template with the layout but will ultimately be addressed later.